spec/migrations/20160115111829_chargeback_rate_detail_currency_not_nil_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+require_migration
+
+describe ChargebackRateDetailCurrencyNotNil do
+  migration_context :up do
+    before(:each) do
+      @cbd = FactoryGirl.create(:chargeback_rate_detail,
+                                :rate     => 8,
+                                :per_time => "hourly",
+                                :per_unit => "megabytes",
+                                :metric   => 'derived_memory_available',
+                               )
+    end
+
+    it "change to default currency" do
+      currency = ChargebackRateDetailCurrency.first.id
```

Comment:
This check assumes the database has a `ChargebackRateDetailCurrency`, which it won't in test.  See above comment.